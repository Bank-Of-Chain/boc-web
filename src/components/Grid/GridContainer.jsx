import React from "react";
import PropTypes from "prop-types";
import { makeStyles } from "@material-ui/core/styles";

// === Components === //
import Grid from "@material-ui/core/Grid";

const styles = {
  grid: {
    width: "auto",
  },
};

const useStyles = makeStyles(styles);

export default function GridContainer(props) {
  const classes = useStyles();
  const { children, className, ...rest } = props;
  return (
    <Grid container {...rest} className={classes.grid + " " + className}>
      {children}
    </Grid>
  );
}

GridContainer.defaultProps = {
  className: "",
};

GridContainer.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string,
};
