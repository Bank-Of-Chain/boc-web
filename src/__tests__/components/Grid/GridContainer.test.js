import React from 'react'
import { render } from '@testing-library/react'
import GridContainer from '@/components/Grid/GridContainer'

test('GridContainer Component Render', () => {
  const { asFragment } = render(<GridContainer />)
  expect(asFragment()).toMatchSnapshot()
})
