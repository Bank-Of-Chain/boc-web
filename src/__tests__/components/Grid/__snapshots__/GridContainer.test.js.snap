// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridContainer Component Render 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root makeStyles-grid-1  MuiGrid-container"
  />
</DocumentFragment>
`;

exports[`TestComponent Component Render 1`] = `
<DocumentFragment>
  <span>
    TestComponent
  </span>
</DocumentFragment>
`;
